### DISCUSSION TITLE
Assembly of single cells with spades

### DISCUSSION THREAD

**Discussion by beacamara** [ablab/spades#1399] [Q&A]
### Description of bug

We have used SPADES to assembly Single-cell data (using --sc parameter). In some of the resulting contigs I have detected terminal redundancy with a size of 127 bp. Surprisingly, this is the the same size of one of the k-mers lengths used by spades during the assembly. Is this an error of spades?

![image](https://github.com/user-attachments/assets/855ec385-a367-4545-aaee-d5e9e07e14a0)

### spades.log

[spades.log](https://github.com/user-attachments/files/17476249/spades.log)

### params.txt

[params.txt](https://github.com/user-attachments/files/17476252/params.txt)

### SPAdes version

SPAdes version: 3.13.0

### Operating System

Linux-5.14.21-150400.24.46-default-x86_64-with-glibc2.31

### Python Version

Python version: 3.9.12

### Method of SPAdes installation

conda

### No errors reported in spades.log

- [X] Yes

**Comment by asl** [comment]
> I have detected terminal redundancy with a size of 127 bp.

It is perfectly fine if the sequence is circular. Have you checked this?
